src/managing-workflow/platform-monitoring.md

```diff
@@ -1,3 +1,58 @@
 # Platform Monitoring
 
-Platform monitoring is a work in progress. If you wish to follow the progress you can do so [here](https://github.com/deis/monitor).
\ No newline at end of file
+## Description
+With the release of Workflow Beta4 we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 4 components:
+* [Telegraf](https://docs.influxdata.com/telegraf/v0.12/) - Metrics collection daemon written by team behind InfluxDB.
+* [InfluxDB](https://docs.influxdata.com/influxdb/v0.12/) - Time series database
+* [Grafana](http://grafana.org/) - Graphing tool for time series data
+* [Stdout-Metrics](https://github.com/deis/stdout-metrics) - Tool for consuming metrics via standard out and forwards them to InfluxDB
+
+## Architecture Diagram
+
+```
+┌────────┐                  
+│ Router │                  
+└────────┘                  
+     │                      
+     │                      
+     ▼          ┌──────────┐
+┌─────────────┐         ┌─────────┐     │ stdout   │
+│ HOST        │         │ fluentd │────▶│  metrics │
+│  Telegraf   │───┐     └─────────┘     └──────────┘
+└─────────────┘   │                           │     
+│                           │     
+┌─────────────┐   │                           │     
+│ HOST        │   │    ┌───────────┐          │     
+│  Telegraf   │───┼───▶│ InfluxDB  │◀─────────┘     
+└─────────────┘   │    └───────────┘                
+│          ▲                      
+┌─────────────┐   │          │                      
+│ HOST        │   │          │                      
+│  Telegraf   │───┘    ┌──────────┐                 
+└─────────────┘        │ Grafana  │                 
+└──────────┘                 
+```
```

Comment:
I pretty much get this diagram, but I think it could be cleaned up a bit:

- What is the line between the top Telegraf box and fluentd box for?
- Can you make the line between the InfluxDB and Grafana box have an arrow (I'm assuming it would be bidirectional)
- What is the line under the bottom Telegraf box for?